lib/find-visualstudio.js

```diff
@@ -456,6 +457,8 @@ class VisualStudioFinder {
       return 'v142'
     } else if (versionYear === 2022) {
       return 'v143'
+    } else if (versionYear === 2025) {
+      return 'v143'  // Fix this when Visual Studio 2025 ships
```

Comment:
Is these VS 2025 changes required now? Shall we add these when VS 2025 is actually released?

Reply:
That is probably a question for @StefanStojanovic.  I am not a Windows user but if Windows 2025 is available on GitHub Action, it will not be long before new issues are opened about using it.  The great news is that it seems to work out-of-the box and in some tests is quite a bit faster.  The only incompatibility appears in our integration tests because we are still many months away from having a Visual Studio 2025.

Reply:
The way I see it, Windows and VS versions are not connected. We are combining them for our tests (Win2019 + VS2019 and Win2022 + VS2022), but that is not mandatory.

@cclauss can you please explain this part of the comment: _**These tests will fail on Windows 2025 without these mods because Windows 2025 currently ships with Visual Studio 2022.**_ Does that mean that only changing the `yml` files results in test failures?

Reply:
If you make the changes to the YAML files but do not make the changes to `lib/find-visualstudio.js` then the Windows 2025 tests will fail.

Reply:
> Windows and VS versions are not connected.

My sense is that `lib/find-visualstudio.js` assumes that they are connected.

Reply:
Nice!  Thanks @legendecas you are correct.